support-frontend/assets/helpers/internationalisation/country.ts

```diff
@@ -602,7 +602,7 @@ const countries: Record<IsoCountry, string> = {
 export type UsState = $Keys<typeof usStates>;
 export type CaState = $Keys<typeof caStates>;
 export type AuState = $Keys<typeof auStates>;
-export type StateProvince = UsState | CaState | AuState;
```

Comment:
Similar to above - what happened to Canada and Australia? 😃

Reply:
Hmmm, unsure where this is used either, the checkout does work with state selector CAD+AUD states, possibly set in redux but does not appear to be called upon.

Reply:
For both the above - it's because the type of `CaState`, `AuState` and `UsState` is actually `string`.

I've left it for now, but will follow up in a PR immanently to fix it.